src/TimelineID/TimelineId.tsx

```diff
@@ -1,47 +1,50 @@
-import React, { FunctionComponent } from "react";
+import React, { FunctionComponent , useState, useEffect} from "react";
 import styles from './TimelineID.module.scss';
 import { useParams, Link } from "react-router-dom";
 import { VerticalTimeline, VerticalTimelineElement } from 'react-vertical-timeline-component';
 import 'react-vertical-timeline-component/style.min.css';
 
-interface Rover {
-    name: string;
-    information: string;
-    imageUrl: string;
+
+export interface TimelineEvent {
+    imageUrl?: string | undefined;
+    heading: string;
+    body: string;
+    date: string;
 }
 
+
 interface RoverName{
     roverName: string;
 }
 
+/* istanbul ignore next */
+const fetchTimelines = async (roverName: string): Promise<TimelineEvent[]> => {
+    const response= await fetch(`https://mars-mission-backend-staging.herokuapp.com/api/rovers/${roverName}/timeline`);
```

Comment:
actually hang on.. this needs to be using an environment variable!
(other work in progress is calling it `REACT_ENV_API_URL`, so it you do the same it should all be smooth!)

just dump `REACT_ENV_API_URL=http://localhost:3001` in a `.env` file